src/components/button-group/ButtonGroup.scss

```diff
@@ -60,30 +60,30 @@
         }
     }
 
-    > .btn:first-child {
+    > .bdl-Button:first-child {
         border-top-left-radius: 4px;
         border-bottom-left-radius: 4px;
     }
 
-    > .btn:last-child {
+    > .bdl-Button:last-child {
         border-top-right-radius: 4px;
         border-bottom-right-radius: 4px;
     }
 
-    > .btn.is-selected {
+    > .bdl-Button.bdl-is-selected {
         z-index: 2; /* place on top of siblings */
         color: $bdl-gray-80;
         background-color: $bdl-gray-10;
         border-color: $bdl-gray-62;
         box-shadow: none;
     }
 
-    > .btn:focus {
+    > .bdl-Button:focus {
         z-index: 3; /* place on top of all other buttons for accessibility */
     }
 
-    &.is-disabled {
-        > .btn {
+    &.bdl-is-disabled {
```

Comment:
Should the state `is-disabled` be prefixed with namespace?

Reply:
Yup, state classes also require prefixes: https://confluence.inside-box.net/display/ETO/CSS+Class+Naming#CSSClassNaming-Stateofacomponent. We previously identified `.is-disabled` as a particularly generic class name that has caused issues for Platform customers: https://cloud.app.box.com/notes/477304138591?s=gvy1ikzuzz2s2p42z1ttkhswo20miecp